marlowe-dashboard-client/src/Dashboard/Lenses.purs

```diff
@@ -22,15 +19,13 @@ import Data.Lens.Record (prop)
 import Data.Map (Map, insert, lookup)
 import Data.Maybe (Maybe(..))
 import Data.Symbol (SProxy(..))
-import InputField.Types (State) as InputField
 import Marlowe.PAB (PlutusAppId)
 import Template.Types (State) as Template
-import Types (WebData)
-import WalletData.Types (WalletDetails, WalletInfo, WalletLibrary)
-import WalletData.Validation (WalletIdError, WalletNicknameError)
+import WalletData.Types (State) as WalletData
+import WalletData.Types (WalletDetails)
 
-_walletLibrary :: Lens' State WalletLibrary
-_walletLibrary = prop (SProxy :: SProxy "walletLibrary")
+_walletDataState :: Lens' State WalletData.State
+_walletDataState = prop (SProxy :: SProxy "walletDataState")
```

Comment:
Why walletDataState instead of walletState?. Data and State kind of refer to the same thing. In my head, Data is best used when the values don't change and State if they change over time.

Reply:
Because it's the state of the `WalletData` module. `WalletData` isn't a good name, but I haven't come up with a better one yet. I would just use `Wallet`, but that will clash with the generated PAB modules. The other natural option would be `Contact`, but that is very easy to confuse with `Contract` here, so I'm trying to avoid it. :confused: